env_docker_ext_test.go

```diff
@@ -5,14 +5,15 @@ package e2e_test
 
 import (
 	"bytes"
-	"io/ioutil"
+	"io"
 	"net/http"
 	"path/filepath"
 	"testing"
 
 	"github.com/efficientgo/core/testutil"
 	"github.com/efficientgo/e2e"
 	e2edb "github.com/efficientgo/e2e/db"
+	e2e2 "github.com/efficientgo/e2e/monitoring"
```

Comment:
I think it would be easier to read if we use `e2emon` as in the other parts of the code - wdyt?